### PR TITLE
Add deletion of orphaned Fra1 floating IPs

### PR DESCRIPTION
**Pull request by Rozzii**
This commit adds openstack commands to check and if possible
delete orphaned floating IPs from Fra1 region.

### CODE REVIEW

File: jenkins/scripts/integration_delete.sh [hunk 1]
```diff
@@ -40,6 +40,9 @@ then
   echo "Executer floating IP ${TEST_EXECUTER_FIP_ID} is deleted."
 fi
 
+# Check and delete orphaned floating IPs
+openstack floating ip list --status "DOWN" --column "ID" -f json | jq --raw-output '.[]."ID"'| xargs -0 openstack floating ip delete
```
Comment (lentzi90) [inline jenkins/scripts/integration_delete.sh:44]: nit
```suggestion
openstack floating ip list --status "DOWN" --column "ID" -f json | jq --raw-output '.[]."ID"' | xargs -0 openstack floating ip delete
```

**Review by lentzi90 — COMMENTED** [review]
One nit, otherwise this is good!

**Comment by Rozzii** [comment]
> One nit, otherwise this is good!
> 
> Edit: Should we run this in both Kna1 and Fra1 or just Fra1?

Only for Fra1, we don't use floating IPs for the test jobs in Kna1. Floating IPs are created automatically in Fra1 because otherwise the static workers that live in Kna1 would not be able to connect to Fra1 ephemeral workers.

**Comment by lentzi90** [comment]
> Only for Fra1, we don't use floating IPs for the test jobs in Kna1. Floating IPs are created automatically in Fra1 because otherwise the static workers that live in Kna1 would not be able to connect to Fra1 ephemeral workers.

Should we not move it into the if statement above then? Now I think it will run on both Fra1 and Kna1.

**Comment by Rozzii** [comment]
> > Only for Fra1, we don't use floating IPs for the test jobs in Kna1. Floating IPs are created automatically in Fra1 because otherwise the static workers that live in Kna1 would not be able to connect to Fra1 ephemeral workers.
> 
> Should we not move it into the if statement above then? Now I think it will run on both Fra1 and Kna1.

Sorry you are right it was a mistake I wanted to put it into the condition, good catch thanks.

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Review by smoshiur1237 — APPROVED** [review]
lgtm

**Pull request merged**